packages/terra-responsive-element/src/terra-dev-site/doc/responsive-element/ResponsiveElement.1.doc.mdx

```diff
@@ -27,12 +37,20 @@ This component requires the following peer dependencies be installed in your app
 | react | ^16.8.5 |
 | react-dom | ^16.8.5 |
 
-
-
 <!-- AUTO-GENERATED-CONTENT:END -->
 
 ## Component Features
 
-* [Cross-Browser Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#cross-browser-support)
-* [Responsive Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#responsive-support)
-* [Mobile Support](https://github.com/cerner/terra-ui/blob/master/src/terra-dev-site/contributing/ComponentStandards.e.contributing.md#mobile-support)
+* [Cross-Browser Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#cross-browser-support)
+* [Responsive Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#responsive-support)
+* [Mobile Support](https://engineering.cerner.com/terra-ui/contributing/terra-ui/component-standards#mobile-support)
+
+[View component source code](https://github.com/cerner/terra-core/tree/master/packages/terra-responsive-element)
+
+## Examples
+<BreakpointExample title="Responsive Breakpoints" description="An example demonstrating the callback events during a breakpoint change." />
+<ResizeExample title="Responsive Resizing" description="An example demonstrating the callback events during a resize." />
+<UncontrolledBreakpointExample title="[Deprecated] Uncontrolled Example" description="An example of an uncontrolled implementation of the ResponsiveElement." />
```

Comment:
Thoughts on removing this example since it will soon be deprecated?

Reply:
done here https://github.com/cerner/terra-core/pull/2864/commits/91a0e30e1e2746bc2e0363a1af4afec879906d7f